package.json

```diff
@@ -101,6 +101,7 @@
     "eslint-plugin-chai-friendly": "^0.5.0",
     "eslint-plugin-cup": "^2.0.2",
     "eslint-plugin-flowtype": "^4.6.0",
+    "eslint-plugin-header": "just-at-uber/eslint-plugin-header#ef55ce547ecb6801da326c75374711f0865a57a6",
```

Comment:
Pinned to fork of eslint-plugin-header repo. Waiting for maintainer to merge PR. Can see open PR here:
https://github.com/Stuk/eslint-plugin-header/pull/32